src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,13 @@
+const ArticleCard = ({article}) =>{
+    return( 
+    <div>
+        <h3>{article.title}</h3>
+        <h4>{article.topic}</h4>
+        <p>Author: {article.author}</p>
+        <p>Post date: {article.created_at}</p>
+        <p>Up votes: {Number(article.votes)}</p>
+        <img src={article.article_img_url}/>
+    </div>)
+}
```

Comment:
Accessibility wise your img should have an alt tag and I would swap div for section where possible, semantically is easier to understand what it is.